### DISCUSSION TITLE
Community chat

### DISCUSSION THREAD

**Discussion by rapha8l** [sammy-ette/Hilbish#202] [General]
Hi,

Is there somewhere (irc, discord, etc.) to discuss about Hilbish ?
And if there's none, what would you think of creating one ?
Thanks !

**Comment by sammy-ette** [comment]
the link is on the readme (the chat badge). it's actually my personal community server since hilbish isn't that big enough for a specific server

**Reply by rapha8l** [reply]
thanks !
